lib/aws-record/record.rb

```diff
@@ -57,13 +83,25 @@ def self.included(sub_class)
       sub_class.send(:include, DirtyTracking)
       sub_class.send(:include, Query)
       sub_class.send(:include, SecondaryIndexes)
+      if Aws::Record.extends_record?(sub_class)
+        inherit_track_mutations(sub_class)
+      end
+    end
+
+    def self.extends_record?(klass)
```

Comment:
Since this isn't private, lets add `# @api private` to it.

Reply:
Done!